examples/metering.rs

```diff
@@ -132,27 +140,31 @@ fn main() -> anyhow::Result<()> {
             );
         }
         Err(_) => {
-            println!("Calling `add_one` failed: not enough gas points remaining.");
+            println!("Calling `add_one` failed.");
+
+            // Because the last needed more than the remaining points, we should have an error.
+            let remaining_points = get_remaining_points(&instance);
+
+            match remaining_points {
+                MeteringPoints::Remaining(..) => {
+                    bail!("No metering error: there are remaining points")
+                }
+                MeteringPoints::Depleted(remaining, missing) => println!(
+                    "Not enough points remaining: {:?} points remaining but need {:?}",
+                    remaining,
+                    remaining + missing
```

Comment:
I'd avoid exposing `missing` to the user, as this is not what is missing for the completion of the execution but just what is missing for the next step. I don't think it is very useful.